### PR TITLE
Upgrade typescript version to 4.4.2

### PR DESCRIPTION
**Pull request by vovakulikov**

### CODE REVIEW

File: client/web/src/search/home/DynamicWebFonts/useDynamicWebFonts.ts [hunk 1]
```diff
@@ -34,16 +16,6 @@ declare global {
         variant?: CSSOMString
         featureSettings?: CSSOMString
     }
-
-    interface FontFaceSet extends Iterable<FontFace> {
```
Comment (valerybugakov) [inline client/web/src/search/home/DynamicWebFonts/useDynamicWebFonts.ts:38]: Oh, that's awesome!

**Review by valerybugakov — APPROVED** [review]
Thanks for keeping our TS version on edge!

**Review by eseliger — APPROVED** [review]

**Comment by eseliger** [comment]
@vovakulikov on a TS upgrade streak!

**Review by felixfbecker — APPROVED** [review]
Let's go!!

**Pull request merged**